### ISSUE TITLE
Unable to parse phone with a country code with 3 length

### ISSUE THREAD

**Issue by aitboudad** [catamphetamine/libphonenumber-js#173]
The following number return an empty object instead of `{country: "MA", phone: "659777777"}`

```ts
parse('+212659777777')
```

**Comment by catamphetamine** [comment]
Follow the instructions in the Bug Reporting section, then reopen the issue.
https://github.com/catamphetamine/libphonenumber-js#bug-reporting

**Comment by aitboudad** [comment]
@catamphetamine added the Reproduction example + Google's libphonenumber demo.

**Comment by catamphetamine** [comment]
@aitboudad Oh, you even made a demo yourself, that's good.
The news is that there was a bug introduced recently which prevented some of the countries from being parsed.
It was discovered by @atsu85 today and has just been fixed.
I tested it - the previous version doesn't parse your number while the latest (fixed) version does.
So, anyone who installed `libphonenumber-js` in the past 4 days is affected by the bug.
I'll delete those buggy versions from npm but that's as much as can be done.
With everyone locking their dependencies nowadays I guess we can still expect analogous bug reports in this repo and dependants of it.
I'm gonna create a separate issue for that and leave it open.
